extensions/wikia/Wall/WallThread.class.php

```diff
@@ -109,13 +113,25 @@ private function loadReplyIdsFromDB($master = false) {
 			$list[] = $row->comment_id;
 		}
 
-		$this->setReplies( $list );
+		$lastId = end( $list );
+
+		return empty( $lastId ) ? $list :
+			array_merge( $list, $this->getReplyIdsFromDB( $dbr, $lastId ) );
```

Comment:
Please avoid recursion when possible.
Usually it consumes a lot more of memory.